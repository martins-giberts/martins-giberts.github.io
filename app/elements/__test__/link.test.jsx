/* globals describe, it, expect */

import React from 'react'
import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json'

import Component from '../link/index'

describe('elements/Link test', () => {
  it('should render defaults without throwing an error', () => {
    const wrapper = shallow(<Component/>)
    expect(toJson(wrapper)).toMatchSnapshot()
  })

  it('should render custom without throwing an error', () => {
    const wrapper = shallow(
      <Component href='#' target='_self'>
        <span>Link</span>
      </Component>
    )
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
