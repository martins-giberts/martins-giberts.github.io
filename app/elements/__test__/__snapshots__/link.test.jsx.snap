// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`elements/Link test should render custom without throwing an error 1`] = `
<a
  className="link"
  href="#"
  target="_self"
>
  <span>
    Link
  </span>
</a>
`;

exports[`elements/Link test should render defaults without throwing an error 1`] = `
<a
  className="link"
  href="#"
  target="_blank"
/>
`;
